src/components/JobLogs/JobLogs.js

```diff
@@ -89,4 +92,7 @@ class JobLogs extends Component {
   }
 }
 
-export default compose(withJobLogsCellsFormatter, withJobLogsSort)(JobLogs);
+export default compose(
+  withJobLogsCellsFormatter,
+  withJobLogsSort,
+)(JobLogs);
```

Comment:
Oh, do you need to `injectIntl` here?

Reply:
No, it is provided in `withJobLogsCellsFormatter`. So there is no need to inject it twice.